examples/libf3d/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 project(libf3d-examples)
 
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.19)
```

Comment:
why ?

Reply:
because reading json needs 3.19  
Strictly speaking, it's only needed for the SDK for plugins though.

Reply:
indeed, so this change is not needed